meta-jsdelivr/recipes-jsdelivr/jsdelivr-scripts/files/jsdelivr-startWorld.sh

```
@@ -1,50 +1,177 @@
 #!/bin/bash
 
-source /etc/muid.data
+# Source shared utilities (includes LED functions and boot device detection)
+source /usr/bin/jsdelivr-utils.sh
+
+# Source muid.data (may not exist on very first boot before firstBoot.sh completes)
+if [ -f /etc/muid.data ]; then
+    source /etc/muid.data
```

Shellcheck (0.11.0)

[info] Not following: /etc/muid.data was not specified as input (see shellcheck -x).

(SC1091)